packages/vite/src/client/client.ts

```diff
@@ -74,7 +74,7 @@ async function handleMessage(payload: HMRPayload) {
             ) as HTMLLinkElement[]
           ).find((e) => e.href.includes(path))
           if (el) {
-            const newPath = `${path}${
+            const newPath = `${base}${path.slice(1)}${path}${
```

Comment:
Looks like a typo here: both `path.slice(1)` and `path` is too much!

Reply:
Good catch, I think my tests locally may have been ignore the extra path.  I'll test that shortly and push up the change.